test/utils_test.py

```diff
@@ -1,28 +1,12 @@
-import unittest
-
-# from u.winrm.winrm_session import winrm_session
-
-
-class TestStringMethods(unittest.TestCase):
-    # run = winrm_session('192.168.130.129', '\Marco Mendez', 'Control123')
-    # def test_set_execution_policy_to_unrestrict(self):
-    #     expect_result = 0
-    #     actual_result = self.run.set_execution_policy_to_unrestrict
-    #     self.assertEqual(expect_result,actual_result)
-
-    def test_upper(self):
-        self.assertEqual('foo'.upper(), 'FOO')
-
-    # def test_isupper(self):
-    #     self.assertTrue('FOO'.isupper())
-    #     self.assertFalse('Foo'.isupper())
-    #
-    # def test_split(self):
-    #     s = 'hello world'
-    #     self.assertEqual(s.split(), ['hello', 'world'])
-    #     # check that s.split fails when the separator is not a string
-    #     with self.assertRaises(TypeError):
-    #         s.split(2)
-
-if __name__ == '__main__':
-    unittest.main()
\ No newline at end of file
+from utils.winrm.winrm_session import winrm_session
+from vm_classes.vm_window_7 import vm_window_7
+
+win7 = vm_window_7('192.168.130.129', '\Marco Mendez', 'Control123')
+win8 = vm_window_7('192.168.130.129', '\Marco Mendez', 'Control123')
```

Comment:
are you using vm_window_7 to create a win8 instance? in all case should not be better to have a vm_window generic for windows?

Reply:
Sorry, I made some changes, and that line of code, should not be there.
I will fix...